docs/spec/src/protocol/contracts.md

```diff
@@ -68,21 +70,19 @@ $$
 numChunks \cdot (1 - \frac{100}{\gamma * codingRate}) \geq maxNumOperators
 $$
 
-where $\gamma$ = confirmationThreshold - adversaryThreshold
+where $\gamma = confirmationThreshold - adversaryThreshold$
 
 ### EigenDARelayRegistry
 
-Contains EigenDA network registered Relays’ Ethereum address and DNS hostname or IP address. `BlobCertificate`s contain `relayKey`(s), which can be transformed into that relay’s URL by calling [relayKeyToUrl](https://github.com/Layr-Labs/eigenda/blob/77d4442aa1b37bdc275173a6b27d917cc161474c/contracts/src/core/EigenDARelayRegistry.sol#L35).
+Contains EigenDA network registered Relays’ Ethereum address and DNS hostname or IP address. `BlobCertificates` contain `relayKeys`, which can be transformed into that relay’s URL by calling [relayKeyToUrl](https://github.com/Layr-Labs/eigenda/blob/77d4442aa1b37bdc275173a6b27d917cc161474c/contracts/src/core/EigenDARelayRegistry.sol#L35).
 
 ### EigenDADisperserRegistry
 
 Contains EigenDA network registered Dispersers’ Ethereum address. The EigenDA Network currently only supports a single Disperser, hosted by EigenLabs. The Disperser’s URL is currently static and unchanging, and can be found on our docs site in the [Networks](https://docs.eigenda.xyz/networks/mainnet) section.
 
-## Deployments
-
-<!-- TODO: add deployed contract addresses table -->
-
 ## Governance Roles
 
 <!-- TODO: import from https://www.notion.so/eigen-labs/EigenDA-V2-Governance-17513c11c3e0806999cfe5e8b9bf7e6a -->
 <!-- Do we want to make public everything in that doc?? -->
+
+TODO
```

Comment:
intentional?

Reply:
Yes that's old already, and it was designed by arch. Pretty sure @pakim249CAL is changing some of that. This section will need to be updated by him eventually once our governance roles are stable and well defined.